README.md

```diff
@@ -6,13 +6,15 @@
 [ci-style-status]: https://github.com/cesium/CeSIUMpeloMundo/actions/workflows/style.yml/badge.svg
 [ci-style-workflow]: https://github.com/cesium/CeSIUMpeloMundo/actions/workflows/style.yml
 
-# CeSIUM pelo Mundo
+# :world_map: CeSIUM pelo Mundo
 
-> :world_map: A world map with the places we've been
+> A world map with the places we've been
```

Comment:
I prefer it the other way around because is like the short description of the repositories, that's why most repositories do it this way.

e.g.

- https://github.com/cesium/cesium.link
- https://github.com/cesium/quizium
- https://github.com/cesium/yearbook